src/pages/Main/Footer/FootSlide.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { BiRadioCircle, BiRadioCircleMarked } from 'react-icons/bi';
+import './FootSlide.scss';
+
+export class FootSlide extends Component {
+  constructor() {
+    super();
+    this.state = {
+      slideNumber: 0,
+      slideList: [],
+    };
+  }
+
+  componentDidMount() {
+    this.startSlide();
+    fetch('http://localhost:3000/data/MainSlide.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          slideList: data,
+        });
+      });
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.startSlide);
+  }
+
+  startSlide = () => setInterval(this.autoSlide, 2500);
+  endSlide = () => clearInterval(this.startSlide);
+
+  autoSlide = () => {
+    const { slideNumber, slideList } = this.state;
+    if (slideNumber >= slideList.length - 1) {
+      this.setState({ slideNumber: 0 });
+    } else {
+      this.setState({ slideNumber: this.state.slideNumber + 1 });
+    }
+  };
+
+  slideHandler = e => {
+    const value = e.currentTarget.value;
+    this.setState({ slideNumber: Number(value) });
+  };
+
+  render() {
+    const { slideList, slideNumber } = this.state;
+    return (
+      <div className="FootSlide">
+        <ul
+          className="slide"
+          style={{ transform: `translateX(${-100 * slideNumber}vw)` }}
+        >
+          {slideList.map(slide => (
+            <li className="slideList" key={slide.id} value={slide.id}>
+              <article className="slideContent">
+                <img src={slide.icon} alt={slide.alt} />
+                <h2>{slide.description}</h2>
+                <p> {slide.user}</p>
+              </article>
+            </li>
+          ))}
```

Comment:
```suggestion
          {slideList.map(({id,icon,alt,description,user})=> (
            <li className="slideList" key={id} value={id}>
              <article className="slideContent">
                <img src={icon} alt={alt} />
                <h2>{description}</h2>
                <p> {user}</p>
              </article>
            </li>
          ))}
```
인자로 받을 때 바로 구조분해 할 수도 있습니다.
별개로 데이터구조에서 alt라는 걸 받는 것 같은데 일반적으로 백엔드에서 주는 데이터는 alt를 포함해서 주진 않습니다.
목업데이터도 백엔드랑 소통하면서 실제 백엔드 api의 데이터형식과 동일하게 맞추는게 좋습니다!

Reply:
오 저부분 궁금했었는데 감사합니다!
넵 데이터 부분은 수정이 필요할것 같습니다.